### ISSUE TITLE
Failed to resolve

### ISSUE THREAD

**Issue by umesh7995** [jhonnyx2012/HorizontalPicker#34]
**Failed to resolve: devs.mulham.horizontalcalendar:horizontalcalendar:1.3.2**
This error getting after Jcenter() deprecated, 

- mavenCentral()
- gradlePluginPortal()

Even though i had replaced with these two, same error

**Comment by dearsikandarkhan** [comment]
Yes . experiencing the same issue. Any equivalent alternative libraries you got @umesh7995 ?

**Comment by umesh7995** [comment]
https://www.youtube.com/watch?v=-7DKf1Q3pME&t=426s 
No, I don't find any other dependency. So I'm using my horizontal picker.
@Sikandarkhan

**Comment by dearsikandarkhan** [comment]
this is is interesting. Let me check the implementation.
